base/variables.scss

```diff
@@ -10,13 +10,15 @@ $header-font-family: $gibson;
 // Font Sizes
 $base-font-size: 18px;
 $quote-font-size: $base-font-size * 0.9;
-$h1-font-size: 3.25em;
-$h2-font-size: 2.5em;
-$h3-font-size: 1.75em;
-$h4-font-size: 1.2em;
+$h1-font-size: 2.5em;
+$h1-dark-font-size: 1.875em;
+$h2-font-size: 1.5em;
+$h3-font-size: 1em;
+$h4-font-size: 1.125em;
```

Comment:
Right now, `h4` is only being used in one place – under the `h1` on the homepage. It's also a lighter font-weight, so don't feel this is the greatest offense. If we use it elsewhere, I think we should be more careful. 

It might be a pain in the ass, but we could also change this to the `h3` styles and change all current `h3`s (much more frequently used) to `h4`s